src/MongoDB.Driver.Core/Core/Bindings/CoreSession.cs

```diff
@@ -271,7 +271,7 @@ public void CommitTransaction(CancellationToken cancellationToken = default(Canc
 
                 try
                 {
-                    var firstAttempt = CreateCommitTransactionOperation();
+                    var firstAttempt = CreateCommitTransactionOperation(IsFirstCommitARetry());
```

Comment:
It's up to you, but in my opinion, `A` letter looks not needed here.

Reply:
I initially thought the `A` looked weird too, but then for some reason it looked weirder to me without the `A`, and now I agree with you that the `A` is not necessary. 😆 TLDR: I agree; making the change!